content/it/platform_as_a_service.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Platform as a Service (PaaS)
+status: Completed
+category: Tecnologia
+---
+
+## Cos'è
+Una Platform as a Service, o PaaS, è una piattaforma esterna ai team di sviluppo per distribuire ed eseguire le proprie applicazioni. Heroku, Cloud Foundry, App Engine sono esempi di offerte PaaS.
+
+## Quali problematiche affronta
+Per sfruttare i modelli nativi del cloud come i [microservizi](/microservices/) o le applicazioni distribuite, i team delle operation e gli sviluppatori devono essere in grado di scaricare dalle proprie board una quantità significativa di operazioni e lavori di manutenzione. Questi includono attività come il provisioning dell'infrastruttura, la gestione dell'individuazione dei servizi e il bilanciamento del carico, e la scalabilità delle applicazioni.
+
+## In che modo aiuta
+Un servizio PaaS fornisce strumenti di infrastruttura agli sviluppatori di applicazioni in modo completamente automatizzato. Consente loro di comprendere e preoccuparsi meno dell'infrastruttura stessa e di dedicare più tempo e sforzi alla scrittura del codice dell'applicazione. Fornisce inoltre una base di monitoraggio e osservabilità per aiutare i team ad assicurarsi che le proprie applicazioni siano sane. 
```

Comment:
`health-check` è sinonimo di "raggiungibilità" e, per estensione, di funzionamento di un'applicazione, server, etc...
```suggestion
Un servizio PaaS fornisce strumenti di infrastruttura agli sviluppatori di applicazioni in modo completamente automatizzato. Consente loro di comprendere e preoccuparsi meno dell'infrastruttura stessa e di dedicare più tempo e sforzi alla scrittura del codice dell'applicazione. Fornisce inoltre una base di monitoraggio e [osservabilità](/observability/) per aiutare i team ad assicurarsi che le proprie applicazioni siano funzionanti. 
```

Reply:
sì, anche in italiano un "sistema sano" o "un'applicazione sana" è qualcosa che funziona nel modo in cui dovrebbe funzionare - quindi non solo funzionare, ma funzionare bene e come dovrebbe. ma va benissimo il tuo suggerimento, probabilmente dà adito a meno fraintendimenti!